datadog_checks_base/tests/conftest.py

```diff
@@ -9,5 +9,5 @@
 
 @pytest.fixture(scope="session")
 def socks5_proxy():
-    with docker_run(compose_file=os.path.join(HERE, "compose", "socks5-proxy.yaml")):
+    with docker_run(compose_file=os.path.join(HERE, "compose", "socks5-proxy.yaml"), sleep=5):
```

Comment:
Should we use a wait condition instead ?

Reply:
cc @FlorianVeaux what do you recommend checking here?